pydicom/config.py

```diff
@@ -38,6 +46,32 @@ def reset_data_element_callback():
     data_element_callback_kwargs = {}
 
 
+def DS_numpy(use_numpy=True):
+    """Set whether multi-valued elements with VR of DS will be numpy arrays
+
+    Parameters
+    ----------
+    use_numpy : bool
+        ``True`` to read multi-value DS elements as `:class:~numpy.ndarray`
+        ``False`` to read multi-valued DS data elements as type
+           `:class:~python.mulitval.MultiValue`
+        Note: once a value has been accessed, changing this setting will
+        no longer change its type
+
+    Raises
+    ------
+    ValueError
+        If use_DS_Decimal and use_numpy are both True.
+    """
+
+    global use_DS_numpy
+
+    if use_DS_decimal is True and use_numpy is True:
```

Comment:
I was thinking of the possibility of combining the multiple DS cases into one config item... so just trying to future-proof here in case later they became strings or something.  But I think it is worked out now so can remove the `is True`.